CORRELATION_CONTEXT_HTTP_HEADER_FORMAT.md

```diff
@@ -0,0 +1,64 @@
+# Correlation Context HTTP Header Format
+
+A correlation context header is used to pass the name-value context properties (also known as tags) for the trace. This is a companion header for the `Trace-Context`. The values are intended for control of logging systems and should be passed along to any child requests. 
+
+# Format
+
+## Header name
+
+`Correlation-Context`
```

Comment:
I would argue a bit more about the name, I like the simplicity of the format.

Reply:
Do you want to argues as part of this PR or as part of #13?